.github/workflows/goth-unstable.yml

```diff
@@ -0,0 +1,76 @@
+name: goth-unstable
```

Comment:
The two workflow files, `goth-stable.yml` and `goth-unstable.yml`, mostly overlap. There's no way to have the common parts in one place?

Reply:
I'm not aware of any official solution to re-use an existing workflow, you can take a look at this thread for some more context: https://github.com/actions/runner/issues/748 (there's a custom-built solution linked there, though I'm reluctant about using it for our needs).

With that said, I think I'm going to change the definition of stable and unstable runs to depend on `yagna` and API versions used rather than the declared stability of a given test.

Reply:
question though @zakaprov -> couldn't we parametrize the runs: so that the diference would be on the level of particular (conditional) steps rather than on the level of workflows? e.g. by adding a different `input` for the both scheduled runs? 

something along the lines of:
https://stackoverflow.com/questions/62736315/invoke-github-actions-workflow-manually-and-pass-parameters

Reply:
actually, probably better to link the actual documentation: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs

Reply:
@shadeofblue Unfortunately, it's currently not possible to get the values provided as inputs in a scheduled workflow run (see here: https://github.community/t/workflow-dispatch-and-schedule-with-inputs/131004). However, in our specific case I think we could have a step which conditionally sets env variables depending on whether it's a scheduled workflow or not. I'll try and see whether it's worth a shot.